types/encoding.go

```diff
@@ -115,6 +115,35 @@ func (c *Currency) UnmarshalSia(r io.Reader) error {
 	return nil
 }
 
+// HumanString prints the Currency using human readable units. The unit used
+// will be the largest unit that results in a value greater than 1. The value is
+// rounded to 4 significant digits.
+func (c Currency) HumanString() string {
```

Comment:
added tests. Decided to leave `siac` as it is - displaying numbers through siac and through logs do not naturally require the same formatting. While they use the same formatting right now, we may in the future decide that different levels of precision, etc. are required for each situation.

In light of that, I'm going to leave them separate.